README.md

```diff
@@ -83,4 +83,4 @@ Navigate to **OAuth & Permissions** in your app configuration and click **Add a
 
 ```
 https://3cb89939.ngrok.io/slack/oauth_redirect
-```
```

Comment:
This one might be a questionable change but I think **it'll be alright** for the time being 😳! 😂